view/frontend/web/js/view/cart/google-pay.js

```diff
@@ -176,7 +176,11 @@ define([
             this.setPaymentMethodNonce(nonce);
             this.isPlaceOrderActionAllowed(true);
             saveShipping.saveShippingInformation();
-            this.placeOrder();
+
+            // Getting All Response Then call PlaceOrder function
+            setTimeout(() => {
+                this.placeOrder();
+            }, 1000);
```

Comment:
Just wondering if there is a better method of waiting for All Responses
Save for above

Reply:
@mstein-ck, Sometimes, it not save the shipping address to quote, So i added settimeout to there. It throws an error message like shipping address is required or missing.